recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: optv
+  version: 0.2.3
+
+source:
+  git_url: https://github.com/OpenPTV/openptv.git
```

Comment:
Please, do not use git, use tarballs instead
Please see:
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarball#build-from-tarballs-not-repos